core/uemis-downloader.c

```diff
@@ -76,6 +76,14 @@ static int dive_to_read = 0;
 
 static int max_deleted_seen = -1;
 
+// Linked list to remember already executed divespot download requests
+struct divespot_downloads{
+	int divespot_id;
+	uint32_t dive_site_uuid;
+	struct divespot_downloads *next;
+};
+struct divespot_downloads *divespot_downloads = NULL;
```

Comment:
Since it is only used in this translation unit - make it of static linkage?

Reply:
Right.